ui/.vscode/settings.json

```diff
@@ -4,5 +4,7 @@
     "cSpell.enabled": false,
     "i18n-ally.localesPaths": "src/assets/i18n",
     "i18n-ally.dirStructure": "file",
-    "i18n-ally.keystyle": "nested"
+    "i18n-ally.keystyle": "nested",
+    "editor.indentSize": "tabSize",
+    "editor.tabSize": 2

```

Comment:
was it a mistake to push this?

Reply:
Yes (and no... because we should have common global settings for indentation).